### ISSUE TITLE
`superposer` cli referening commented-out args

### ISSUE THREAD

**Issue by ljmartin** [volkamerlab/opencadd#123]
Hi OC, 
Line https://github.com/volkamerlab/opencadd/blob/7f23496a2f1fb0fe8a78251398255dea6aeff8f2/opencadd/structure/superposition/cli.py#L108 references `args.method_options`, but in `parse_cli()`, `method_options` has been commented out. This prevents use of the cli. 

I've got a fresh install from mamba. Looks like removing the `**args.method_options` won't work either:

```
(compchem) ljmartin@Lewiss-MacBook-Pro protein_alignment % superposer struc1.pdb struc2.pdb
 ┌─┐┬ ┬┌─┐┌─┐┬─┐┌─┐┌─┐┌─┐┌─┐┬─┐
 └─┐│ │├─┘├┤ ├┬┘├─┘│ │└─┐├┤ ├┬┘
 └─┘└─┘┴  └─┘┴└─┴  └─┘└─┘└─┘┴└─
 Brought to you by @volkamerlab

Warning: importing 'simtk.openmm' is deprecated.  Import 'openmm' instead.
Traceback (most recent call last):
  File "/Users/ljmartin/miniconda3/envs/compchem/bin/superposer", line 10, in <module>
    sys.exit(main())
  File "/Users/ljmartin/miniconda3/envs/compchem/lib/python3.9/site-packages/opencadd/structure/superposition/cli.py", line 107, in main
    result, *_empty = align(
  File "/Users/ljmartin/miniconda3/envs/compchem/lib/python3.9/site-packages/opencadd/structure/superposition/api.py", line 45, in align
    result = aligner.calculate([reference.models[0], mobile.models[0]])
  File "/Users/ljmartin/miniconda3/envs/compchem/lib/python3.9/site-packages/MDAnalysis/core/topologyattrs.py", line 203, in stub_method
    raise NoDataError(message)
MDAnalysis.exceptions.NoDataError: Structure.models() not available; this requires models
```
Not sure how to fix that one, so any comment would be appreciated!

**Comment by dominiquesydow** [comment]
Hi @ljmartin,

That you for raising this issue and my apologies for such a late reply.

@corey-taylor and @pipaj97 do you have any ideas and/or could look into this, please?
Will this be maybe fixed by https://github.com/volkamerlab/opencadd/pull/117? If I remember correctly the PR also affects the CLi.

**Comment by pipaj97** [comment]
This should be fixed by PR #117.
The PR will soon be merged into the master branch.

**Comment by ljmartin** [comment]
No problem - thanks @pipaj97 and @dominiquesydow !
